plugins/out_calyptia/calyptia.c

```diff
@@ -1057,7 +1089,11 @@ static struct flb_config_map config_map[] = {
      "Pipeline ID for calyptia core traces."
     },
 #endif
-
+    {
+     FLB_CONFIG_MAP_BOOL, "register_retry_on_flush", "true",
+     0, FLB_TRUE, offsetof(struct flb_calyptia, register_retry_on_flush),
+     "Retry agent registration on flush if failed on init."
```

Comment:
For some of the tests we may need to set this false but I think auto-retry is a good default.